components/brave_wallet_ui/common/slices/endpoints/token_balances.endpoints.ts

```diff
@@ -446,6 +450,46 @@ export const tokenBalancesEndpoints = ({
           id: [arg.coin, arg.chainId, arg.address].join('-')
         }
       ]
+    }),
+    getBitcoinBalances: query<
+      BitcoinBalances,
+      { accountID: BraveWallet.AccountId }
+    >({
+      queryFn: async (arg, { endpoint }, extraOptions, baseQuery) => {
+        try {
+          const { data: api } = baseQuery(undefined)
+          const { balance, errorMessage } =
+            await api.bitcoinWalletService.getBalance(arg.accountID)
+
+          if (errorMessage || balance === null) {
+            console.log(`getBitcoinBalances error: ${errorMessage}`)
+            return {
+              error: errorMessage || 'Unknown error'
+            }
+          }
```

Comment:
We can re-use the logic in the catch block if we throw instead
```suggestion
          if (errorMessage || balance === null) {
            throw new Error(errorMessage || 'Unknown error')
          }
```

Reply:
Updated!